iothub/service/src/Configurations/Configuration.cs

```diff
@@ -101,7 +101,7 @@ internal Configuration()
         /// Custom Configuration Metrics
         /// </summary>
         [JsonProperty(PropertyName = "metrics", NullValueHandling = NullValueHandling.Ignore)]
-        public ConfigurationMetrics Metrics { get; internal set; }
+        public ConfigurationMetrics Metrics { get; set; }
 
         /// <summary>
```

Comment:
Can you add a E2E test for this change since this is public API I assume setting this will be reflected on the hub.